api/v1alpha1/sysbench_types.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The xridge kubestone contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// SysbenchSpec defines the desired state of Sysbench
+type SysbenchSpec struct {
+	// Image defines the sysbench docker image used for the benchmark
+	Image ImageSpec `json:"image"`
+
+	PodConfigurationSpec `json:",inline"`
```

Comment:
I shouldn't  comment this here, but just realized:
In Fio, this is not inlined, while in Iperf3 it is inlined.  We should make this consistent across the three benchmarks.

@apollolemonte  / @julio-santiesteban / @wgershwitz : What is your preference? Inlining or having a dedicated var?

Reply:
The CR tries to describe a Pod/Job with benchmark specific parameters. If we do not inline PodConfigurationSpec, then we should move ImageSpec into PodConfigurationSpec to be consistent with Kubernetes API. All in all, I prefer inlining.